GVRf/Framework/src/org/util/jassimp/AiConfigOptions.java

```diff
@@ -0,0 +1,745 @@
+/*
+ * $Revision$
+ * $Date$
+ */
+package org.util.jassimp;
+
```

Comment:
Is this file from third-party or created by you? I don't see any valid file header here.